src/hooks/bookmarks/use-all-bookmarks.ts

```diff
@@ -20,7 +20,7 @@ interface UseAllBookmarksResult
  * Handles fetching and storing all bookmarks with React Query.
  */
 const useAllBookmarks = ({
-	enabled,
+	enabled = true,
```

Comment:
Should this be connected to the authentication state?

Reply:
It is! 😊 The `enabled` option passed to `useQuery` below is set using this new option and `accessToken` retrieved from `useAuthentication`.